limine/limine.cfg

```
@@ -0,0 +1,24 @@
+INTERFACE_BRANDING=Void Linux
+TIMEOUT=8
+TERM_WALLPAPER=boot:///boot/limine/@@SPLASHIMAGE@@
+TERM_WALLPAPER_STYLE=centered
+TERM_BACKDROP=#000000
+
+:@@BOOT_TITLE@@ @@KERNVER@@ @@ARCH@@
+    PROTOCOL=linux
+    KERNEL_PATH=boot:///boot/vmlinuz
+    CMDLINE= root=live:CDLABEL=VOID_LIVE init=/sbin/init ro rd.luks=0 rd.md=0 rd.dm=0 loglevel=4 vconsole.unicode=1 vconsole.keymap=@@KEYMAP@@ locale.LANG=@@LOCALE@@  @@BOOT_CMDLINE@@
+    MODULE_PATH=boot:///boot/initrd
+    COMMENT=Boot a Void linux live system
+    
+
+:@@BOOT_TITLE@@ @@KERNVER@@ @@ARCH@@ (RAM)
+	PROTOCOL=linux
+    KERNEL_PATH=boot:///boot/vmlinuz
+    CMDLINE= root=live:CDLABEL=VOID_LIVE init=/sbin/init ro rd.luks=0 rd.md=0 rd.dm=0 loglevel=4 vconsole.unicode=1 vconsole.keymap=@@KEYMAP@@ locale.LANG=@@LOCALE@@  @@BOOT_CMDLINE@@ rd.live.ram
+    MODULE_PATH=boot:///boot/initrd
+    COMMENT=Boot a Void linux system loaded in RAM
```

```suggestion
    COMMENT=Boot a @@BOOT_TITLE@@ system loaded in RAM
```
perhaps?
(and same for other entry)